drivers/sub8_rdi_dvl/include/rdi_explorer_dvl/driver.hpp

```diff
@@ -182,18 +258,25 @@ class Device {
 
     std::string str = buf.str();
 
-    try {
+    try  // Write heartbeat to serial port
+    {
       size_t written = 0;
-      while (written < str.size()) {
+      while (written < str.size())
+      {
```

Comment:
Are you using any sort of auto formating for this? We should write a script / add a wiki page for auto formating. Here's a [dudes clang formatter config](https://github.com/davetcoleman/roscpp_code_format) for ROS Style Guide